auth/password_authenticator.cc

```diff
@@ -161,6 +175,8 @@ future<> password_authenticator::stop() {
 }
 
 db::consistency_level password_authenticator::consistency_for_user(std::string_view role_name) {
+    // TODO: this is plain dung. Why treat hardcoded default special, but for example a user-created
+    // super user uses plain LOCAL_ONE?
```

Comment:
Agree. We should switch to try-quorum-fallback-to-local for all types of users. But not in this series.